refinery_core/src/runner.rs

```diff
@@ -255,6 +256,7 @@ impl Runner {
             abort_divergent: true,
             abort_missing: true,
             migrations: migrations.to_vec(),
+            migration_table_name: String::from("refinery_schema_history".to_string()),
```

Comment:
I think here we can just do `refinery_schema_history".into()` but can we also put the name as `DEFAULT_TABLE_NAME` on `refinery_core/src/traits/mod.rs`  with the other consts and use it here?